_sass/custom/custom.scss

```
@@ -16,6 +16,11 @@
 	content: counter(heading)"." counter(subheading)"  ";
 	counter-increment: subheading;
 }
+#text-only-logo {
+    font-weight: bold;
+    font-size: 45px;
+    color: $gap-light-blue;
```

Did you perhaps mean this?
```suggestion
    color: $gap-blue;
```